suites/google/googleServiceAccountKeyTest.js

```diff
@@ -235,6 +247,12 @@ Scenario('SA key removal job test: remove expired creds @reqGoogle', async (fenc
   console.log('Clean up expired Service Account keys');
   bash.runJob('google-manage-keys-job');
```

Comment:
is this another instance where we need to remove the `-job`?

Reply:
I have this tiny change in a separate PR that miraculously passed:
https://github.com/uc-cdis/gen3-qa/pull/219/files